spec/project_spec.rb

```diff
@@ -492,10 +497,12 @@ class InvalidErrorHandler; end
       project_instance.decision_service.set_forced_variation(project_config, 'test_experiment_with_audience', 'test_user', 'variation_with_audience')
       variation_to_return = project_instance.decision_service.get_forced_variation(project_config, 'test_experiment', 'test_user')
       allow(project_instance.decision_service.bucketer).to receive(:bucket).and_return(variation_to_return)
+      sleep 0.1
```

Comment:
I am not sure if we need to put a sleep before **allow**.

Reply:
there is a get_forced _varitation before that.  the sleep is to give the thread a chance to run.